src/Service/MailTemplateHelper.php

```diff
@@ -260,7 +260,7 @@ public function getTemplateFile(MailTemplate $mailTemplate): string
 
     public function getTemplates(string $type): array
     {
-        return $this->mailTemplateRepository->findBy(['type' => $type]);
+        return $this->mailTemplateRepository->findBy(['type' => $type, 'isArchived' => 0], ['name' => 'ASC']);
```

Comment:
Can we use `false` here (rather than `0`)?

Reply:
Yep, changed.